modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -459,3 +459,207 @@ bin/neo4j-admin database backup --to-path=azb://myStorageAccount/myContainer/myD
 ----
 ======
 =====
+
+
+[role=label--new-2025.04]
+[[diff-backup-as-parent]]
+=== Perform a differential backup using the `--prefer-diff-as-parent` option
+
+When taking a differential backup with the `--type=DIFF` option, the parent, by default, is the *most recent non-empty* backup in the directory.
+In some cases, you may prefer to use the *latest differential* backup as the parent, even if there is a more recent full backup.
```

Comment:
What are these cases? If we don't specify, then maybe this part doesn't add much value. So, either we say something like:
```suggestion
However, in some cases, such as ...., you may prefer to use the *latest differential* backup as the parent, even if there is a more recent full backup.

```
or directly say how to use it like:

```
However, you can also use the *latest differential* backup as the parent, even when there is a more recent full backup, by setting the option `--prefer-diff-as-parent` to `true`.